data/getir.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Getir",
+  "url": "https://getir.com/it/",
+  "career_page_url": "https://apply.workable.com/getir/",
+  "type": "Product",
+  "categories": ["B2C"],
```

Comment:
```suggestion
  "categories": ["cloud_software"],
```

Reply:
Sorry, I suggested the wrong code. @napolux

Reply:
👯‍♂️

Reply:
So what's the final version of the file I have to commit? I'm a bit confused now :D